frontendUiTable/build.gradle

```diff
@@ -2,7 +2,10 @@ dependencies {
   vavr()
 
   implementation project(':backendApi')
-  implementation project(':backendRoot')
+  implementation(project(':backendRoot')) {
+    exclude(group: 'org.slf4j', module: 'slf4j-api')
```

Comment:
Hmm I think this shouldn't be necessary once the slf4j-api version is reconciled with what IntelliJ pulls in; see the comments to build.gradle